model/client4_test.go

```diff
@@ -56,3 +57,26 @@ func TestClient4CreatePost(t *testing.T) {
 	_, resp := client.CreatePost(post)
 	assert.Equal(t, http.StatusOK, resp.StatusCode)
 }
+
+func TestClient4SetToken(t *testing.T) {
+	expected := NewId()
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		authHeader := r.Header.Get(HEADER_AUTH)
+
+		token := strings.Split(authHeader, HEADER_BEARER)
+
+		if len(token) < 2 {
+			t.Errorf("wrong authorization header format, got %s, expected: %s %s", authHeader, HEADER_BEARER, expected)
+		}
+
+		assert.Equal(t, expected, strings.TrimSpace(token[1]))
+	}))
+
+	client := NewAPIv4Client(server.URL)
+	client.AuthToken = expected
```

Comment:
Thanks, @atpons -- we're still not actually testing `SetToken` here. Can you change this to read:
```go
	client := NewAPIv4Client(server.URL)
    client.SetToken(expected)
	_, resp := client.GetMe("")
	assert.Equal(t, http.StatusOK, resp.StatusCode)
```

ideally we'd also test `MockSession` for good measure:
```go
	client := NewAPIv4Client(server.URL)
    client.MockSession(expected)
	_, resp := client.GetMe("")
	assert.Equal(t, http.StatusOK, resp.StatusCode)
```

Reply:
I'm sorry. I will fix it now.
Do I need to call `MockSession` on `TestClient4SetToken` and specify **Deprecated** in the comment?
Or do I make 'TestClient4MockSession`?

Reply:
I think good to prepare 2 tests and if `MockSession` will be deprecated, to remove `TestClient4MockSession`.